internal/pkg/deploy/cloudformation/stack/validate_test.go

```diff
@@ -567,7 +569,9 @@ func TestValidateWorkerName(t *testing.T) {
 }
 
 func TestValidateTopicSubscription(t *testing.T) {
-	validTopics := []string{"arn:aws:us-east-1:123456789012:app-env-svc-name", "arn:aws:us-east-1:123456789012:app-env-svc-name2"}
+	validTopics := []string{"arn:aws:sns:us-east-1:123456789012:app-env-svc-name", "arn:aws:sns:us-east-1:123456789012:app-env-svc-name2"}
+	app := "app"
+	env := "env"
 	testCases := map[string]struct {
```

Comment:
We can add one more test case that shows how we behave when one of the `validTopics` cannot be parsed.